src/navigators/LoggedInStack.tsx

```diff
@@ -71,6 +81,11 @@ export function LoggedInStack() {
       return t('root-stack-accepting-invitation', 'Accepting invitation');
     } else if (loadingJoinCircles) {
       return t('root-stack-waiting-for-circle-join', 'Joining Circles');
+    } else if (shouldWaitForOnAppStart) {
+      return t(
+        'root-stack-waiting-for-on-app-session-start',
+        'Processing app session start',
+      );
```

Comment:
Not for this PR, and possibly not ever, but something to think about: what should SDK consumers expect if their func errors out and never calls `resumeAppSession`?  More than the decision, I think something that's key is that we document this so devs can write defensive code accordingly.  My gut is to be somewhat like a Promise ctor, and document the danger of an `onAppSessionStart` that never calls `resumeAppSession` -> it will _never_ progress.

Reply:
Making calling `resumeAppSession` required to resume the app progression is a drawback of the callback approach. I'll update the doc in a follow-up PR.